syncd/syncd_flex_counter.cpp

```diff
@@ -804,14 +924,16 @@ void FlexCounter::collectCounters(
 
         // Get queue stats
         sai_status_t status = -1;
-//        TODO: replace if with get_queue_stats_ext() call when it's fully supported
-//        Example:
-//        sai_status_t status = sai_metadata_sai_queue_api->get_queue_stats_ext(
-//                queueId,
-//                static_cast<uint32_t>(queueCounterIds.size()),
-//                queueCounterIds.data(),
-//                m_statsMode,
-//                queueStats.data());
+#if 0
+        TODO: replace if with get_queue_stats_ext() call when it is fully supported
```

Comment:
Why use #if 0?
If you are going to use it, add '//' in L#928 and remove L#929 entirely

Reply:
It looks prettier in vim